integration/src/test/java/edu/unc/lib/boxc/integration/factories/FileFactory.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.boxc.integration.factories;
+
+import edu.unc.lib.boxc.model.api.DatastreamType;
+import edu.unc.lib.boxc.model.api.objects.FileObject;
+import edu.unc.lib.boxc.model.fcrepo.ids.DatastreamPids;
+import edu.unc.lib.boxc.model.fcrepo.services.DerivativeService;
+import org.apache.commons.io.FileUtils;
+import edu.unc.lib.boxc.model.api.rdf.IanaRelation;
+import org.apache.jena.vocabulary.DCTerms;
+
+import static edu.unc.lib.boxc.model.api.DatastreamType.TECHNICAL_METADATA;
+import static edu.unc.lib.boxc.model.api.xml.NamespaceConstants.FITS_URI;
+import static org.apache.jena.rdf.model.ResourceFactory.createResource;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * @author snluong
+ */
+public class FileFactory extends ContentObjectFactory {
+    public static final String FILE_FORMAT = "fileFormat";
```

Comment:
Sorry, I was thinking it would be FILE_FORMAT_OPTION and the others would stay IMAGE_FORMAT, etc, so that it was more clear that FILE_FORMAT_OPTION was a key in the options object, while IMAGE_FORMAT was a value.